pynicotine/search.py

```diff
@@ -532,17 +532,10 @@ def process_search_request(self, searchterm, user, searchid, direct=False):
 
         self.core.send_message_to_peer(user, message)
 
-        if direct:
-            log.add_search(
-                _("User %(user)s is directly searching for \"%(query)s\", returning %(num)i results"), {
-                    'user': user,
-                    'query': searchterm_old,
-                    'num': numresults
-                })
-        else:
-            log.add_search(
-                _("User %(user)s is searching for \"%(query)s\", returning %(num)i results"), {
-                    'user': user,
-                    'query': searchterm_old,
-                    'num': numresults
-                })
+        log.add_search(
+            _("User %(user)s is %(mode)s for \"%(query)s\", found %(num)i results"), {
```

Comment:
I would prefer to keep two separate strings. I'm not sure if the change will work in every language.